### PR TITLE
Add bugzilla references

### PR DESCRIPTION
**Pull request by j-mracek**

### CODE REVIEW

File: dnf-behave-tests/features/microdnf/upgrade-pkgspec.feature [hunk 1]
```diff
@@ -1,7 +1,7 @@
 @not.with_os=rhel__eq__8
 Feature: Upgrade RPMs by pkgspec
 
-
+@bz1905471
```
Comment (m-blaha) [inline dnf-behave-tests/features/microdnf/upgrade-pkgspec.feature:4]: You cannot tag Background section.

**Comment by m-blaha** [comment]
The upgrade tests are passing, but there are two `module/switch.feature` scenarios that are failing on RHEL:
```
  @bz1792020 @bz1809314 @feature_file_name:features/module/switch.feature
  Scenario: Switch a module to stream that was only enabled  # features/module/switch.feature:26
    Given I use repository "dnf-ci-fedora-modular"           # features/steps/repo.py:138
    When I execute dnf with args "module enable nodejs:10"   # features/steps/cmd.py:106
    Then the exit code is 0                                  # common/output.py:33
    And Transaction is following                             # features/steps/transaction.py:138
      | Action               | Package   |
      | module-stream-enable | nodejs:10 |
    And modules state is following                           # features/steps/module.py:101
      | Module | State   | Stream | Profiles |
      | nodejs | enabled | 10     |          |
    When I execute dnf with args "module switch-to nodejs:8" # features/steps/cmd.py:106
    Then the exit code is 0                                  # common/output.py:33
      Assertion Failed: Command has returned exit code 1: dnf -y --installroot=/tmp/dnf_ci_installroot_i50nuhg7 --releasever=29 --setopt=module_platform_id=platform:f29 --disableplugin='*' module switch-to nodejs:8

Last Command: dnf -y --installroot=/tmp/dnf_ci_installroot_i50nuhg7 --releasever=29 --setopt=module_platform_id=platform:f29 --disableplugin='*' module switch-to nodejs:8

Last Command stdout:
Dependencies resolved.

Last Command stderr:
The operation would result in switching of module 'nodejs' stream '10' to stream '8'
Error: It is not possible to switch enabled streams of a module.
It is recommended to remove all installed content from the module, and reset the module using 'dnf module reset <module_name>' command. After you reset the module, you can install the other stream.

    And Transaction is following                             # None
      | Action               | Package        |
      | module-stream-switch | nodejs:10 -> 8 |
    And modules state is following                           # None
      | Module | State   | Stream | Profiles |
      | nodejs | enabled | 8      |          |

  @bz1792020 @bz1809314 @feature_file_name:features/module/switch.feature
  Scenario: I can switch a module to stream, install packages from a new module and keep installed profile  # features/module/switch.feature:46
    Given I use repository "dnf-ci-fedora-modular"                                                          # features/steps/repo.py:138
    Given I use repository "dnf-ci-fedora"                                                                  # features/steps/repo.py:138
    When I execute dnf with args "module install nodejs:8/minimal"                                          # features/steps/cmd.py:106
    Then the exit code is 0                                                                                 # common/output.py:33
    And Transaction contains                                                                                # features/steps/transaction.py:153
      | Action                 | Package                                       |
      | install-group          | nodejs-1:8.11.4-1.module_2030+42747d40.x86_64 |
      | module-profile-install | nodejs/minimal                                |
    And modules state is following                                                                          # features/steps/module.py:101
      | Module | State   | Stream | Profiles |
      | nodejs | enabled | 8      | minimal  |
    When I execute dnf with args "module switch-to nodejs:10"                                               # features/steps/cmd.py:106
    Then the exit code is 0                                                                                 # common/output.py:33
      Assertion Failed: Command has returned exit code 1: dnf -y --installroot=/tmp/dnf_ci_installroot_bz5osyoo --releasever=29 --setopt=module_platform_id=platform:f29 --disableplugin='*' module switch-to nodejs:10

Last Command: dnf -y --installroot=/tmp/dnf_ci_installroot_bz5osyoo --releasever=29 --setopt=module_platform_id=platform:f29 --disableplugin='*' module switch-to nodejs:10

Last Command stdout:
Dependencies resolved.

Last Command stderr:
The operation would result in switching of module 'nodejs' stream '8' to stream '10'
Error: It is not possible to switch enabled streams of a module.
It is recommended to remove all installed content from the module, and reset the module using 'dnf module reset <module_name>' command. After you reset the module, you can install the other stream.

    And Transaction contains                                                                                # None
      | Action               | Package                                        |
      | upgrade              | nodejs-1:10.11.0-1.module_2200+adbac02b.x86_64 |
      | upgrade              | npm-1:10.11.0-1.module_2200+adbac02b.x86_64    |
      | module-stream-switch | nodejs:8 -> 10                                 |
    And modules state is following                                                                          # None
      | Module | State   | Stream | Profiles |
      | nodejs | enabled | 10     | minimal  |
    When I execute dnf with args "module switch-to nodejs:8"                                                # None
    Then the exit code is 0                                                                                 # None
    And Transaction contains                                                                                # None
      | Action               | Package                                       |
      | downgrade            | nodejs-1:8.11.4-1.module_2030+42747d40.x86_64 |
      | downgrade            | npm-1:8.11.4-1.module_2030+42747d40.x86_64    |
      | module-stream-switch | nodejs:10 -> 8                                |
    And modules state is following                                                                          # None
      | Module | State   | Stream | Profiles |
      | nodejs | enabled | 8      | minimal  |

```

File: dnf-behave-tests/features/microdnf/upgrade-pkgspec.feature (hunk 1 above)
Comment (j-mracek) [reply dnf-behave-tests/features/microdnf/upgrade-pkgspec.feature:4]: sorry

Comment (j-mracek) [reply dnf-behave-tests/features/microdnf/upgrade-pkgspec.feature]: Fixed

**Comment by m-blaha** [comment]
the bugzilla referencing tags are correct.

**Pull request merged**